test/e2e/fips.go

```diff
@@ -18,7 +18,8 @@ const (
 	mFips = "99-master-fips"
 )
 
-var _ = Describe("Validate FIPS Mode", func() {
+// FIPS Mode is set differently and the test needs updating after 2022-04-01 API is active
```

Comment:
Would you please create a workitem for this, so we can add it to the toil and plan it?

Reply:
Already in place [WI](https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/12385409)